manifests/init.pp

```diff
@@ -297,6 +297,7 @@
   $apm_enabled = $datadog_agent::params::apm_default_enabled,
   $apm_env = 'none',
   $apm_non_local_traffic = false,
+  $apm_analyzed_spans = $datadog_agent::params::$apm_analyzed_spans_list,
```

Comment:
We probably want to define types here, given that this is a new parameter:
```
Array[Array[String]] $apm_analyzed_spans = $datadog_agent::params::$apm_analyzed_spans_list
```

Maybe even better:
```
Optional[Array[Tuple[String, 3]]] $apm_analyzed_spans = undef
```

Reply:
Also, you might want to add support for specifying apm analyzed spans via hiera in the manifest as well, you can do this in a future PR if you prefer.

Reply:
Have updated to you recommendations.